pkg/environment/environment.go

```diff
@@ -69,13 +71,49 @@ func RecoverEnvironmentErrors(errch chan<- error) {
 	errch <- nil
 }
 
-// GetInputChannels returns environment variable which contains the input channels
-func GetInputChannels() string {
+// GetInputChannels returns the input channels
+func GetInputChannels() []brokers.ChannelBroker {
```

Comment:
Similar thing here. "GetInputChannels" seems that you're returning only a list of channels